proto/config/netcmn.proto

```diff
@@ -84,11 +84,21 @@ message ipspec {
   string gateway = 5;
   string domain = 6;
   string ntp = 7;
+  repeated string moreNtp = 10;
```

Comment:
How abot calling this more_ntp_servers and adding a comment that ntp+more_ntp_servers is used to specify a set of NTP servers. Also, can these be hostnames or IP addresses? Makes sense to state that in a comment.

Reply:
Done - https://github.com/lf-edge/eve-api/pull/70/files#diff-35ea4ed74707f6ad6d17be169425a7acf4e25e09e9de81a0f7300ed80c5ec656R86